commit c39d0454ec9b703d3540dd10a2e9692f89aa48ab
Author: Ido Schimmel <idosch@mellanox.com>
Date:   Thu Dec 3 12:12:04 2015 +0100

    net: Add support for CHANGEUPPER notifier error injection
    
    Since CHANGEUPPER can now fail, add support for it in the newly
    introduced netdev notifier error injection infrastructure.
    
    Signed-off-by: Ido Schimmel <idosch@mellanox.com>
    Signed-off-by: Jiri Pirko <jiri@mellanox.com>
    Acked-by: Nikolay Aleksandrov <nikolay@cumulusnetworks.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

diff --git a/lib/netdev-notifier-error-inject.c b/lib/netdev-notifier-error-inject.c
index b2b856675d61..13e9c62e216f 100644
--- a/lib/netdev-notifier-error-inject.c
+++ b/lib/netdev-notifier-error-inject.c
@@ -18,6 +18,7 @@ static struct notifier_err_inject netdev_notifier_err_inject = {
 		{ NOTIFIER_ERR_INJECT_ACTION(NETDEV_POST_INIT) },
 		{ NOTIFIER_ERR_INJECT_ACTION(NETDEV_PRECHANGEMTU) },
 		{ NOTIFIER_ERR_INJECT_ACTION(NETDEV_PRECHANGEUPPER) },
+		{ NOTIFIER_ERR_INJECT_ACTION(NETDEV_CHANGEUPPER) },
 		{}
 	}
 };